src/app/docs/components/search/search.module.scss

```diff
@@ -0,0 +1,31 @@
+.search-input {
+  width: 100%;
```

Comment:
Maybe it will be better to make it a little bit shortly and add `border-radius: 9999px` then it will look like our `rounded` buttons style

Reply:
I wouldn't round it that much because it starts to look ugly to me. I think it's better to wait until the designer can help us with that page